apps/src/code-studio/components/SoundListEntry.jsx

```diff
@@ -78,6 +78,7 @@ class SoundListEntry extends React.Component {
         },
         {includeUserId: true}
       );
+      this.props.soundsRegistry.unmuteURLs();
```

Comment:
Do we need to re-mute the URLs in the onEnded callback?

Reply:
Done!